Machine_Learning/Basic_Projects_of_Machine_Learning/Kaggle's Titanic Prediction/Model/README.md

```diff
@@ -0,0 +1,171 @@
+# Project Title : Kaggle's Titanic Prediction
+
+## Abstract/Brief Description about the Project :
+
+![image](https://user-images.githubusercontent.com/63282184/142458721-c0a32098-a527-4941-9ee2-f08ba91af594.png)
+
+
+
+We have access to a bunch of 9 features (numerical, text, categorical). The <b>big challenge</b> with this competition is the size of the data we have. The <b>training set</b> is composed of only <b>891 samples</b>. The testing set is composed of 418 samples. <br>Therefore, the main issue is to design an algorithm which generalizes enough in order to avoid <b>over-fitting</b>. To do so, a bunch of features is generated. Then, an ensemble modeling method with voting is used in order to get the most generalized model.<br><br>
+
+## Dataset :
+
+This is a multi-label classification, with 2 labels:
+
+- 0: passenger did not survive
+- 1: passenger survived
+
+[Kaggle](https://www.kaggle.com) offers 2 datasets:
+- One Training set: 891 passengers whose label is known
+- One Test set (TS0): 418 passengers whose label is unknown
+
+
+
+
+
+## Goal/Aim of the project :
+
+Goal: For each passenger, predict the label (0 or 1).
+
+## Libraries used :
```

Comment:
Provide the libraries you have used here under this section. @ashish-reddy-20-08

Reply:
Done sir , i made the changes

Reply:
1. Write only `sklearn`
2. Include all the libraries which are mentioned in the `requirements.txt` file of your project.

@ashish-reddy-20-08

Reply:
Approved! @ashish-reddy-20-08